### PR TITLE
[mobile] Second pass on device adaptation section

### PR DESCRIPTION
**Pull request by tidoust**
Main changes:
- Improved the introduction
- Dropped the Device Description Repository API: it is neither widespread anymore, nor a good idea
- Moved Media Queries Level 4 and 5 to "in progress" section
- Moved ambient light Media Queries directive from Level 4 to Level 5
- Re-indented the page.

### CODE REVIEW

**Comment by tidoust** [comment]
Wait, I'll add Device Memory and HTTP Client Hints as suggested in #51

**Comment by tidoust** [comment]
@xfq, now done and ready for review.

Please use Squash and merge to get rid of the clumsy history I created in that PR ;)

File: mobile/adaptation.html [hunk 1]
```diff
@@ -1,50 +1,52 @@
...
-
-        <section class="featureset in-progress">
-          <h2>Specifications in progress</h2>
-          <p data-feature="Media Capture Capabilities">The <a data-featureid="getusermedia-cap">Media Capture and Streams</a> API exposes some specific information on capabilities of camera and microphones to make it possible to take advantage of the large variety of media capturing devices provided on mobile phones.</p>
-
-          <div data-feature="CSS-based adaptation">
+        <h2>Well-deployed technologies</h2>
+        <div data-feature="CSS-based adaptation">
+          <p><a data-featureid="mediaqueries">CSS Media Queries</a> defines a mechanism that allows adapting the layout and behavior of a Web page based on some of the characteristics of the device, most notably based on the screen resolution.</p>
+          <p>The <a data-featureid="viewport-units">viewport-relative CSS units <code>vw</code> and <code>vh</code></a> let design layouts that adapt to the dimensions of the viewport, representing a percentage of the current viewport dimensions.</p>
```
Comment (xfq) [inline mobile/adaptation.html]: Do we need to mention `vmin` and `vmax`, too? Or we just remove the specific units, and only use "viewport-percentage lengths" or "viewport-relative CSS units" here?

File: mobile/adaptation.html [hunk 2]
```diff
@@ -1,50 +1,52 @@
...
...t adaptation on mobile devices is to have the server deliver the content that fits the user's device right away. This approach avoids sending content to the client that the client does not need, preserving network bandwidth, and it avoids running the adaptation logic on the client itself, preserving CPU and memory. <strong>Server-side adaptation</strong> used to rely on <em>sniffing</em> the <code>User-Agent</code> string and mapping it to a database of devices to extract relevant capabilities. This mechanism is error-prone and requires continuous maintenance of the database of devices. Through the <a href="http://httpwg.org/http-extensions/client-hints.html">HTTP CLient Hints</a> mechanism, developed by the IETF HTTP Working Group, clients and servers may now opt-in to exchange relevant capabilities for efficient content adaptation. The <a href="https://www.w3.org/webperf/">Web Performance Working Group</a> has started to work on a <a data-featureid="device-memory">Device Memory</a> specification that extends these hints to surface the memory available on the device. The specification also defines an API to surface that information to the Web application running on the client.</p>
```
Comment (xfq) [inline mobile/adaptation.html]: Nit: s/HTTP CLient Hints/HTTP Client Hints/

File: mobile/adaptation.html (hunk 1 above)
Comment (tidoust) [reply mobile/adaptation.html]: That may be too technical for this document, but at the same time, mentioning these additional units does not take much space, so why not. See updated file.

I also improved the description a little bit.

File: mobile/adaptation.html (hunk 2 above)
Comment (tidoust) [reply mobile/adaptation.html]: Good catch, thks!

**Comment by xfq** [comment]
Thank you!

**Pull request merged**